### DISCUSSION TITLE
A GUI Gallery

### DISCUSSION THREAD

**Discussion by kaeland** [rabbibotton/clog#274] [Ideas]
@rabbibotton This link contains a gallery of various GUIs. I saw it on hacker news today and thought I'd share it here. I hope all is well. 

http://toastytech.com/guis/index.html

**Comment by worstname** [comment]
Might be of interest as well, a gallery of "HCI" demos, plenty of interesting ideas for UI: https://jackrusher.com/classic-ux/

**Comment by rabbibotton** [comment]
very good timing - I am working on designs for Builder 2.0 while further polishing the current work on CLOG for mobile releases.

**Comment by kaeland** [comment]
The HN thread may have other helpful sites as well: https://news.ycombinator.com/item?id=40276780

**Reply by kaeland** [reply]
https://terminaltrove.com/

**Reply by kaeland** [reply]
https://inconsolation.wordpress.com/2015/01/19/wpe-and-we-someones-been-reading-my-mind/
